client/app/components/sections/ManageAvailability/ManageAvailabilityContainer.js

```diff
@@ -16,13 +16,15 @@ const ManageAvailabilityContainer = ({
   visibleMonth,
   hasChanges,
   saveInProgress,
+  saveFinished,
   reservedDays,
   blockedDays,
   sideWinderWrapper,
 }) =>
       r(ManageAvailability, {
         hasChanges,
         saveInProgress,
+        saveFinished,
```

Comment:
Yeah, maybe that should be done. Some props are restructured, so I wouldn't start refactoring those here.